README.MD

```diff
@@ -0,0 +1,29 @@
+# Welcome to SC2-AI
+
+<p> Welcome to SC2-AI a program built to play Star Craft 2 an implimentation of ALPHAStar</p>
```

Comment:
It's not really program. It's an environment for training bots for SC2. Also, it's not an implementation of AlphaStar.